src/main/java/com/artipie/settings/cache/CachedFilters.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.settings.cache;
+
+import com.amihaiemil.eoyaml.YamlMapping;
+import com.artipie.ArtipieException;
+import com.artipie.http.filter.Filters;
+import com.artipie.misc.ArtipieProperties;
+import com.artipie.misc.Property;
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.LoadingCache;
+import java.util.Optional;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Implementation of cache for filters using {@link LoadingCache}.
+ *
+ * @since 0.28
+ * @checkstyle DesignForExtensionCheck (500 lines)
+ */
+public class CachedFilters implements FiltersCache {
```

Comment:
Suggest to rename `CachedFilters` to `GuavaFiltersCache`.

Reply:
@dgarus Good idea!
Renamed, please review